packages/protocol/contracts/common/MetaTransactionWallet.sol

```diff
@@ -99,6 +99,32 @@ contract MetaTransactionWallet is
     emit EIP712DomainSeparatorSet(eip712DomainSeparator);
   }
 
+  /**
+   * @notice Returns the struct hash of the MetaTransaction
+   * @param destination The address to which the meta-transaction is to be sent.
+   * @param value The CELO value to be sent with the meta-transaction.
+   * @param data The data to be sent with the meta-transaction.
+   * @param _nonce The nonce for this meta-transaction local to this wallet.
+   * @return The digest of the provided meta-transaction.
+   */
+  function getMetaTransactionStructHash(
```

Comment:
if this is public, per recent zeppelin audit, it should probably be listed in the interface as well :)

Reply:
Yeah I'm not actually sure if it makes that much sense to be public. I think `getMetaTransactionDigest` is enough so we can mark this as internal :-/